lib/aws-rails-provisioner/cdk_builder.rb

```diff
@@ -64,7 +65,7 @@ def _install_dependencies
     def _npm_build
       Dir.chdir(@cdk_dir) do
         puts "running npm run build ..."
-        `npm run build`
+        `npm run build --verbose`
```

Comment:
May want to remove verbose here unless required

Reply:
I left it there because there's no other helpful debugging

Reply:
I'll remove it (assumes that the generated code will have no errors but it did before)